lib/ts/recipe/session/with-jwt/sessionClass.ts

```diff
@@ -55,6 +56,27 @@ export default class SessionClassWithJWT implements SessionContainerInterface {
     getExpiry = (userContext?: any): Promise<number> => {
         return this.originalSessionClass.getExpiry(userContext);
     };
+    getSessionGrants(userContext?: any) {
+        return this.originalSessionClass.getSessionGrants(userContext);
+    }
+    updateSessionGrants(grants: Grant<any>[], userContext?: any): Promise<void> {
+        return this.originalSessionClass.updateSessionGrants(grants, userContext);
+    }
+    shouldRefetchGrant(grant: Grant<any>, userContext?: any): Awaitable<boolean> {
+        return this.originalSessionClass.shouldRefetchGrant(grant, userContext);
+    }
+    fetchGrant(grant: Grant<any>, userContext?: any): Awaitable<void> {
+        return this.originalSessionClass.fetchGrant(grant, userContext);
+    }
+    checkGrantInToken(grant: Grant<any>, userContext?: any): Awaitable<boolean> {
+        return this.originalSessionClass.checkGrantInToken(grant, userContext);
+    }
+    addGrant<T>(grant: Grant<T>, value: T, userContext?: any): Promise<void> {
+        return this.originalSessionClass.addGrant(grant, value, userContext);
+    }
+    removeGrant<T>(grant: Grant<T>, userContext?: any): Promise<void> {
+        return this.originalSessionClass.removeGrant(grant, userContext);
+    }
```

Comment:
Rest of the functions in this file are arrow functions. Please maintain the consistency.